### ISSUE TITLE
Consider building a manylinux wheel for this package

### ISSUE THREAD

**Issue by jamescurtin** [shap/shap#1383]
👋  This package currently has wheels that support windows; would you consider updating the CI/CD pipeline to also build a [manylinux](https://github.com/pypa/manylinux) wheel? This would be useful when deploying production applications that use this package to secured environments that don't include necessary build dependencies (_e.g._ `gcc`).

Thanks for considering this!

**Comment by slundberg** [comment]
I would love to do this, but don't have the bandwidth at the moment. Any help is more than welcome!

**Comment by jamescurtin** [comment]
I can take a stab at it! I notice you use both Travis & Appveyor for CI. Any concern with consolidating onto one platform (Travis)?

**Comment by slundberg** [comment]
Great! No concern consolidating in principle, I originally had both because travis support for windows was poor. If that has changed enough then i would be fine consolidating. Simpler is always better. Just want to make sure that we don't lose the windows builds of course :)

If you need details or assistance just let me know. It will be great to have linux builds!

**Comment by jamescurtin** [comment]
Great—thank you for that info. Do you have an existing test.pypi.org account that I could use for QA? If you could either add the encrypted test PyPi cred to the Travis file or share here, I can handle the rest.

**Comment by jamescurtin** [comment]
Great—thank you for that info. Do you have an existing test.pypi.org account that I could use for QA? If you could either add the encrypted test PyPi cred to the Travis file or share here, I can handle the rest.

**Comment by jamescurtin** [comment]
Great—thank you for that info. Do you have an existing test.pypi.org account that I could use for QA? If you could either add the encrypted test PyPi cred to the Travis file or share here, I can handle the rest.

**Comment by slundberg** [comment]
Sounds great, here is an encrypted upload token for test Pypi:

```
secure: "o9MXDlBV8hGA/JntVE6DYwnzD5XrWnsKcRKn5SJpeZkZ7tNENAMDVCAMrF68Dn3MHJZDrHUruhHCE/BUMWXLQw5yp7d6Rc2rlj3f+EgcpU4iknvZLFRhBNJ6Xxk/8cgjCLwPUsjeKh7Em35otwI/lUsfD0vtKS/Kq6cA+g+WRUSZbQ+3VKTupeIhqBMlB45HMg5iAcHpA7OFncKq8HnVcIKlBxuFr8gVqxEEyvgU0mhaJo6q9I5n+37NKbCZtaPlqMDzB5NVTfeeTaBtq7eHgUihBPuRe/0tJuiN/FRkWGM0ChHujvWFPG9lK/JQb1adZnBEgXvVjq2OA5zQkxcKVZ/CjDBMpCZGphjiBEBS/11fsNw/7OKTvu41dpUc+Vmib812Mnd9QGPWMM43aNO2Ko8dt7lW4lfAHZNhdl64NNvd20/ZU+DftuEedA6BpgEpdrdnkxDddEmjRGJO5K7nsKflD/TCPXysySofbNqXJ5U/XoHyQ7mhuHMnthz9HwcqdfIQJwcVaZ2xP65RBmdAHj/l6JVvR4cAmjXXW0hSXo9YmralRKoUbyVWjqU1V17+HsJguvC6byMKfhbtknAQCE4lMnG9UthcP8ztC1kBJupnzcbiJAqTnMD6i5ejR8CzjBj+iARdpJzpLnOVxrjyvSS47NQDIbflbNnRnAfTKdI="
```

**Comment by cwerner** [comment]
Hi. :wave:

I have also been looking for manylinux wheels for an easier deployment of shap. Was there any progress in this regard?

Cheers

**Comment by jklaise** [comment]
@slundberg what is the status on this one? Happy to help as having no wheels for linux can be a pain. Also tangentially related to this: https://github.com/slundberg/shap/pull/1802.

**Comment by connortann** [comment]
This should be closed by #3028  - we should have manylinux wheels in our imminent next release
